lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/PrintHandler.java

```diff
@@ -569,6 +576,28 @@ public void setCCItemXml(Element itemXml, Element resourceXml, AbstractParser pa
       boolean inline = false;
       String mmDisplayType = null;
       String resourceId = resourceXml.getAttributeValue(IDENTIFIER);
+      log.debug("identifier {} identifierRef {} resourceId {} resourceXml {}", identifier, identifierRef, resourceId, resourceXml);
+
+      // Check if there is any Canvas module metadata in the import
+      Element canvas_module_item = null;
+      boolean open_same_window = true;
+      if ( identifier != null && canvasModuleMeta != null ) {
+
+          Namespace namespace = Namespace.getNamespace("cccv1p0", CANVAS_MODULE_META_NAMESPACE);
+
+          String xpathExpression = "//cccv1p0:item[@identifier='" + identifier + "']";
+
+          List<Element> items = XPathFactory.instance().compile(xpathExpression, Filters.element(), null, namespace).evaluate(canvasModuleMeta);
+
+          log.debug("Found {} items whilst looking for {}", items.size(), xpathExpression);
+          if ( items.size() >= 1 ) {
```

Comment:
```
if (CollectionUtils.isNotEmpty(items)) {
```

Reply:
That seems like a bit of overkill to use an Apache library to avoid checking a list size.